.github/workflows/ci.yaml

```diff
@@ -149,6 +149,11 @@ jobs:
           docker compose -f docker/unit-tests.test.yml \
             up --exit-code-from=sut
 
+      - name: Run exitcode tests
+        run: |
+          docker compose -f docker/exitcode-test.yml \
```

Comment:
the filename is really `exitcode-test.yaml` which causes this test to fail.
suggest renaming the file `yml` to match others.